src/components/POLine/POLineForm.js

```diff
@@ -243,6 +244,12 @@ class POLineForm extends Component {
                     accordionStatus={this.state.sections}
                     onToggle={this.onToggleSection}
                   >
+                    <Accordion
+                      label={<FormattedMessage id="ui-orders.line.accordion.itemDetails" />}
+                      id={ACCORDION_ID.itemDetails}
+                    >
+                      <ItemForm stripes={stripes} />
```

Comment:
I see something different inside component:
```
class ItemForm extends Component {
  static propTypes = {
    stripes: PropTypes.object.isRequired,
    parentResources: PropTypes.shape({
      materialTypes: PropTypes.object.isRequired,
    }).isRequired,
    dispatch: PropTypes.func.isRequired,
    change: PropTypes.func.isRequired,
  }
```

Reply:
Yes, you are absolutely right, thank you so much. Fixed.